### DISCUSSION TITLE
et al. & S.l. : s.n.

### DISCUSSION THREAD

**Discussion by Juelianqvq** [stone-zeng/fduthesis#247] [Q&A] [answered]
参考文献中出现的et al.如果想要显示全部作者应该在哪里修改呢？
然后虽然是国标，但S.l. : s.n.也不想显示
谢谢大大！

**Comment by sikouhjw** [comment, marked as answer]
# bibtex

如果你用的是 `bibtex`，请复制一份 `gbt7714-numerical.bst` 放到工作目录，接着参考 `texdoc gbt7714`：

> 参考文献中出现的et al.如果想要显示全部作者应该在哪里修改呢？

修改 bst 文件的 `max.num.authors`的参数；

> 然后虽然是国标，但S.l. : s.n.也不想显示 谢谢大大！

修改 bst 文件的 `show.missing.address.publisher` 参数。

# biblatex

如果你用的是 `biblatex`，请自行参考 `texdoc biblatex-gb7714-2015`。
